test/test/nativemem/doesmalloc.c

```diff
@@ -0,0 +1,47 @@
+/*
+ * Copyright The async-profiler authors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#include <jni.h>
+#include <stdint.h>
+#include <stdlib.h>
+
+JNIEXPORT jlong JNICALL Java_test_nativemem_CallsMallocCalloc_nativeMalloc(JNIEnv* env, jclass clazz, jlong size) {
```

Comment:
`malloc`, `free` and `realloc` are exposed via `sun.misc.Unsafe`.
If not `calloc`, the test could be implemented fully in Java.

Reply:
Thanks, then I will keep JNI usage for all of them for consistence.